Makefile

```diff
@@ -25,3 +26,9 @@ coverage:
 	PYTHONPATH=$$(pwd) coverage run --include=./* -m unittest
 	coverage html
 	coverage report --fail-under=75
+
+docker-build:
```

Comment:
Unfortunately, this sounds a little bit to me like we are building the pipeline-dsl code and not the ci-cd image.
Maybe we can name it `ci-cd-image-build` and `ci-cd-image-push` or maybe you have a better name for it.